SmartHouseGatewayApp/SmartHouseGatewayApp/Startup.cs

```diff
@@ -61,6 +62,12 @@ public void ConfigureServices(IServiceCollection services)
                     }
                 });
             });
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+
+            var connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING") ?? "data source=.;persist security info=False;Initial Catalog=smarthouse_db;User ID=sa;Password=filipkristo1";
```

Comment:
Please load connection string from Configuration. Something like this
`Configuration.GetConnectionString("DefaultConnection")`